convert2rhel/cert.py

```diff
@@ -75,4 +75,7 @@ def remove(self):
             os.remove(self._target_cert_path)
             loggerinst.info("Certificate %s removed" % self._target_cert_path)
         except OSError as err:
-            loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+            if err.errno == 2:
+                pass
```

Comment:
Instead of just calling `pass` here, shouldn't be better to also add a logging? I mean, the rollback message prior to this function is: 

```console
TASK - [Rollback: Removing installed RHSM certificate] ********************
```

we could output a log like this: 

```python
loggerinst.warning("No RHSM certificates found to be removed.")
pass
```

This way the user knows that is something happening in the function and not only "trying to remove something and silently failing". 

The visualization would be something like: 

```console
TASK - [Rollback: Removing installed RHSM certificate] ********************
WARNING - No RHSM certificates found to be removed.
....
```

Reply:
with this in this Jira ticket one of the acceptance criteria's is not logging an error when the RHSM cert is not installed during rollback. so we don't need to log any messages for this error!